### ISSUE TITLE
[models] Remove unused models

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/django-freeradius#133]
@rohithasrk  please do this after #110 is merged.

I think we can safely remove the following models (and related base models, admin classes, docs references):

- `RadiusGroup`
- `RadiusGroupUsers`
- `RadiusReply`

These models were meant to implement the features that are being introduce in #110. They copied the same implementation used in [OpenWISP User Management System](https://github.com/openwisp/OpenWISP-User-Management-System) but we ended up in a different path, which I hope will be simpler to grasp/read and maintain.
The actual models aren't really doing anything. They just sit there, it's dead code.

**Please perform a search and replace of these keywords and remove everything.**

**Comment by 2stacks** [comment]
RadiusReply is part of the default freeradius schema.  Do you think it
should still be removed?

v/r,

Andrew

**Comment by rohithasrk** [comment]
@2stacks Probably as we aren't using it anywhere as of now, @nemesisdesign is asking to remove.

**Comment by nemesisdesign** [comment]
@2stacks maybe I'm wrong, thansk for pointing out. @rohithasrk we'll have to check and remove only the models that are not part of the default freeradius schema.

**Comment by rohithasrk-bidgely** [comment]
Okay. Got it.

**Comment by rohithasrk** [comment]
Okay. I get it.

**Comment by 2stacks** [comment]
No problem.  I don't find it particularly useful but I can imagine someone
else having an existing freeradius configuration that makes use of it.
